kits/beta-base/contracts/BetaKitBase.sol

```diff
@@ -61,16 +65,42 @@ contract BetaKitBase is KitBase {
 
         acl.createPermission(this, dao, dao.APP_MANAGER_ROLE(), this);
 
-        Voting voting = Voting(dao.newAppInstance(appIds[uint8(Apps.Voting)], latestVersionAppBase(appIds[uint8(Apps.Voting)])));
+        Voting voting = Voting(
+            dao.newAppInstance(
+                appIds[uint8(Apps.Voting)],
+                latestVersionAppBase(appIds[uint8(Apps.Voting)]),
+                new bytes(0),
+                true
```

Comment:
Do you mean the doing an extra `setApp`? Otherwise it's hard to retrieve apps from Kernel. Besides, I see at least `Voting` and `TokenManager` playing a "default" role in Democracy and Multisig DAOs.

Reply:
I agree with @sohkai here, Democracy and Multisig DAOs can also be extended by installing extra instances of those apps. Also there is really no advantage in having them as default apps.

Reply:
Ok, reverting it then, although I still kind of disagree ;-)
- Having extra instances doesn't mean you can't have a default. Generally speaking, it's actually the opposite: IMO default makes sense precisely when you have several options (in the sense that when you have just one option you don't need to define a default). In this particular case, I think that a "Democracy DAO" having a default Voting app is a very natural thing. (Same applies for Multisig)
- The advantage is making it easy to access them with `dao.getApp`, plus the reasoning above.